hack/generate-knative.sh

```diff
@@ -89,7 +85,7 @@ if grep -qw "injection" <<<"${GENS}"; then
   # Clear old injection
   rm -rf ${OUTPUT_PKG}
 
-  ${GOPATH}/bin/injection-gen \
+  go run knative.dev/pkg/codegen/cmd/injection-gen \
```

Comment:
I thought you had to pass the list of Go files to go run...

Reply:
directory is fine